app/views/layouts/application.html.erb

```diff
@@ -8,19 +8,35 @@
     <%= stylesheet_link_tag    'application', media: 'all' %>
     <%= javascript_include_tag 'application' %>
   </head>
-    <nav class="nav">
+    <nav class="nav justify-content-center">
       <% if current_user %>
-      <a class="nav-link" href="#">Home</a>
-      <a class="nav-link" href="#">Friends</a>
-      <a class="nav-link active" href="/car_monthly_mileages/new">Carbon Calculator</a>
-      <a class="nav-link" href="#">Offsets</a>
-      <a class="nav-link" href="#">Resources</a>
-      <a class="nav-link" href="#">Log Out</a>
-    <% else !current_user %>
-      <a class="nav-link" href="#">Log In</a>
+      <div class="btn btn-outline-primary">
+        <a class="nav-link" href="/dashboard">Home</a>
+      </div>
+      <div class="btn btn-outline-primary">
+        <a class="nav-link" href="/friends">Friends</a>
+      </div>
+      <div class="btn btn-outline-primary">
+        <a class="nav-link active" href="/car_monthly_mileages/new">Carbon Calculator</a>
+      </div>
+      <div class="btn btn-outline-primary">
+        <a class="nav-link" href="/offsets">Offsets</a>
+      </div>
+      <div class="btn btn-outline-primary">
+        <a class="nav-link" href="/resources">Resources</a>
+      </div>
+      <div class="btn btn-outline-primary">
+        <a class="nav-link" href="/log_out">Log Out</a>
+      <% else !current_user %>
+      <a class="nav-link" href="/auth/google_oauth2">Log In</a>
     <% end %>
```

Comment:
We need to either remove this else clause or remove the if !current_user in the welcome index page.  Otherwise we will have two login links.  I could make more sense to have it in the nav bar unless we want to have to link somewhere else on the page.